arwen/elrondapi/bigIntOps.go

```diff
@@ -345,35 +362,44 @@ func bigIntStorageLoadUnsigned(context unsafe.Pointer, keyOffset int32, keyLengt
 
 	bytes := storage.GetStorage(key)
 
-	value := bigInt.GetOne(destination)
+	value, err := bigInt.GetOne(destinationHandle)
+	if arwen.WithFault(err, context, runtime.BigIntAPIErrorShouldFailExecution()) {
+		return 0
+	}
 	value.SetBytes(bytes)
 
 	return int32(len(bytes))
 }
 
 //export bigIntGetCallValue
-func bigIntGetCallValue(context unsafe.Pointer, destination int32) {
+func bigIntGetCallValue(context unsafe.Pointer, destinationHandle int32) {
 	bigInt := arwen.GetBigIntContext(context)
 	runtime := arwen.GetRuntimeContext(context)
 	metering := arwen.GetMeteringContext(context)
 
 	gasToUse := metering.GasSchedule().BigIntAPICost.BigIntGetCallValue
 	metering.UseGas(gasToUse)
 
-	value := bigInt.GetOne(destination)
+	value, err := bigInt.GetOne(destinationHandle)
+	if arwen.WithFault(err, context, runtime.BigIntAPIErrorShouldFailExecution()) {
```

Comment:
for this case you need GetOrCreate

Reply:
same for here. created before call of function